### ISSUE TITLE
Link and Checkbox keyboard focus is not visible in IE on high contrast mode

### ISSUE THREAD

**Issue by satbai** [OfficeDev/office-ui-fabric-react#4217]
### Bug Report

- __Package version(s)__: 5.46.0 
- __Browser and OS versions__: IE

#### Describe the issue:
Link and Checkbox components' keyboard focus is not visible on IE11 when on high contrast mode.

It looks like Checkbox focus indicator is invalid in normal mode as well. It turns from grey to black, which is not sufficient focus indicator.

#### Actual behavior:
No keyboard focus indicator
![image](https://user-images.githubusercontent.com/24945574/37173772-5a588326-22c9-11e8-8ee9-a6314e7f1ee7.png)
![image](https://user-images.githubusercontent.com/24945574/37174554-b5a10936-22cb-11e8-8af5-9caa0fb82375.png)

#### Expected behavior:
The link and checkbox on high contrast mode should have some kind of focus indicator.

**Comment by Jahnp** [comment]
Good callout, thanks for the heads-up. @betrue-final-final -- any recommendations on what this should be? @satbai is this something you'd be willing to submit a PR for?

**Comment by satbai** [comment]
@Jahnp Unfortunately I don't think I have time to fix it myself.

**Comment by betrue-final-final** [comment]
I'm not getting a repro for link or for checkbox

![image](https://user-images.githubusercontent.com/20363037/37180180-025a0546-22dd-11e8-9703-6100c3242f58.png)

![image](https://user-images.githubusercontent.com/20363037/37180207-1bcd6aea-22dd-11e8-8bf3-a548928308b5.png)

This is on IE11
![image](https://user-images.githubusercontent.com/20363037/37180249-387d7e32-22dd-11e8-90af-8f7e0040f5ae.png)
